622 quick start data and sparkjobs (#1442)
AbsaOSS/enceladus: _docs/2.0.0/usage/data-quick-start.md

    - '2.0.0'
    - 'usage'
---
{% capture docs_path %}{{ site.baseurl }}/docs/{{ page.version }}{% endcapture %}

## Prerequisites

This quick start guide presumes that you have went through :

- [Menas Quick Start]({{ docs_path }}/usage/menas-quick-start) guide

## Data Quality

Data quality is all about 2 things:

- Spline
- _INFO files

## Spline

> but as Spline is under heavy development, we will postpone extensive documentation about it. Enceladus currently runs with version 0.3.X and it works fine out of the box. Spline 0.3.X just needs to be deployed next to Menas.

Reviewer: **Question:** I don't think that Spline 0.3.x has to be "deployed". Perhaps only MongoDb structures needs to be created?

Author: Spline creates the mongo structure as it goes. And it does not have to be deployed to run as per the next paragraph.

Reviewer: Actually yeah, The Spline war has to be deployed... And configured both in Menas and Enceladus
Maybe worth clarifying?

Author: In a quick start? I would rather not. In Deployement process `docs/2.0.0/deployment` sure thing. Can add it in this PR if you don't mind. Then maybe reference it here.

Reviewer: Agreed - with the `docs/2.0.0/deployment` enhancement and referencing here.

Data for Spline are recorded even if the Spline UI is not up and running. This means they can be viewed later without the need to care bout it now.

More about spline at [Spline](https://github.com/AbsaOSS/spline) Github.

## _INFO files

> `_INFO` files our our way of tracking where the data came from and how much is there. It checks mainly that we did not loose any data on the way to standardization and conformance. All this made possibleby [Atum](https://github.com/AbsaOSS/atum). `_INFO` files should be placed withing the source directory together with the raw data.

Reviewer: ```suggestion
`_INFO` files is our way of tracking where the data came from and how much data is there. It checks mainly that no data were lost on the way to and through standardization and conformance. All this is made possible by [Atum](https://github.com/AbsaOSS/atum). `_INFO` files needs ti be placed within the source directory together with the raw data.
```

> `_INFO` file is our way of tracking where the data came from and how much data is there. It checks mainly that no data were lost on the way to and through standardization and conformance. All this is made possible by [Atum](https://github.com/AbsaOSS/atum). `_INFO` files need to  be placed within the source directory together with the raw data.

Reviewer: ```suggestion
`_INFO` file is our way of tracking where the data came from and how much data is there. It checks mainly that no data were lost on the way to and through standardization and conformance. All this is made possible by [Atum](https://github.com/AbsaOSS/atum). `_INFO` file needs to  be placed within the source directory together with the raw data.
```
